lib/generators/active_agent/templates/initializer.rb

```diff
@@ -0,0 +1,3 @@
+# frozen_string_literal: true
+
+ActiveAgent.load_configuration(Rails.root.join('config', 'active_agent.yml'))
```

Comment:
I assumed this should be in the initializer because this is in the Readme, and the existing code was calling methods that don't exist anymore.